classes/player.rb

```diff
@@ -0,0 +1,26 @@
+class Player
+  attr_reader :name, :symbol, :log, :winner
+
+  def initialize (name, symbol)
+    @name = name
+    @symbol = symbol
+
+    @log = []
+    @winner = false
+  end
+
+  def make_move (posibilities)
+    @log.push(*posibilities)
+    @winner = am_i_a_winner
+  end
+
+  private
+
+  def am_i_a_winner
+    @log.each do |position|
+      return true if @log.count(position) == 3
+    end
+
+    false
+  end
```

Comment:
In ruby there is a convention of ending the method name with a `?` if it returns a boolean (and if it makes sense ofcourse). It could be something like `def is_winner?`, also accomplishing the names as verbs

Also, the return inside the loop is redundant, you can remove it.
And in a scenario like this you could use another enumerable method besides `each`
```suggestion
  def is_winner?
    @logs.any? do |log|
       @logs.count(log) == 3
     end
  end
```